export const papers = [
    {
        heading: "Unravelling the basis of assembly, resilience and stability of the gut microbiome communications network",
        desc: `The gut microbiome is not simply a conglomeration of isolated microbes, but a gigantic communications network made up of multiple and diverse lineages of microbes. Interactions (communications) within these lineages are the key to maintaining intra-microbiome and host-microbiome homeostasis and is a marker of our health and well-being. However, while innumerable studies have associated specific microbiome summary statistics and certain members of the gut microflora to with health and/or disease, many aspects of how the gut microbiome communications network is assembled are still a mystery. Under this research theme, we aim to utilize data-driven investigations of a combination of microbiome properties, ranging from taxxa abundances, microbial strain-interactions and (predictive) growth rate profiling of specific dominant microbes constituting our gut communities to understand the language of host-microbiome and intra-microbiome communications. Specifically, we aim to understand how the different members of the gut microbiome assemble in specific configurations to result in a global microbiome assembly; what factors (microbial or otherwise) determines the stability and resilience of this assembly? We also aim to understand how these assembly processes are influenced by our ethnicities, life-style, geography, age and other demographics.`
    },
    {
        heading: "Understanding the tripartite interactions between microbiome, host demographics and external environments/interventions like diet",
        desc: `The human microbiome is one of the key omnipresent ‘organs’ of our body with three functional facets. The first is that of a “sensor”, where it “senses” the alterations and the deterioration of host physiology and acts and modifies itself accordingly, making it an excellent diagnostic target. The second is that of a direct “modulator”, where in the microbiome members either positively or negatively affect host homeostasis, where in these microbiome members can be utilized for diagnostics as well as formulating microbiome-derived therapeutics. The third crucial facet of the microbiome is that of a “transducer”, where in it can either positively or negatively modulate the therapeutic effect of other external interventions like diet, medications, immune-therapies on host health. This aspect of the microbiome has immense applicability in designing microbiome-based stratification strategies that can increase the efficacies of different interventions in a personalized manner. However, despite recent studies demonstrating the role of the microbiome as a transducer of external interventions, the mediating microbiome members have shown wide-spread study-specific variations. This facet forms the basis of this theme. Here, we aim to understand the effect of baseline gut microbiome properties on responses to interventions (specifically focusing on dietary interventions for cardiometabolic disorders and aging), medications (specifically associated with age) and cancer therapies and link these variations to host demographic variation patterns. The goal is to unravel environment-microbiome-host health links that not only enable stratification of different groups of individuals towards a given intervention but also enable the effect of life-style changes on the gut microbiome mediated susceptibilities to different diseases. `
    },
    {
        heading: "Data-driven functional metagenomics of the human microbiome for better mechanistic insights into disease-microbiome interactions and better microbiome-derived disease diagnostics",
        desc: `This theme has two major objectives. First is to investigate and address the basis of inconsistencies in the microbiome alteration patterns associated with the same disorder across studies and design better microbiome-derived diagnostic classifiers. Variations in microbiome alterations have been associated with age, region and socio-economic status of the study population. One of the approaches to bypass the above limitations move away from investigating alterations based on the taxonomic identities of the microbes to performing large-scale functional/metabolic characterizations of all microbiome members and subsequently profiling the functional/metabolic capabilities of the identified altered microbiome members as the diagnostic members. Under this theme, we aim to investigate the above aspect by performing functional genomic characterization of gut microbiome members and utilize these to build microbiome-derived diagnostic classifiers for different diseases and disorders. The second objective is to investigate these alterations and strive to delineate the microbiome alterations into cause and effect classes.`
    },
    {
        heading: "Exploring the human microbiome besides the gut",
        desc: `The gut harbours the largest microbiome of our body, and consequently has been the focus of a large majority of microbiome-host interaction studies. However, microbiomes at several other body sites have also been implicated in different diseases and retaining host homeostasis in these sites. One of the key body sites in this regard has been the oral micro-environment which is not only associated with diseases pertaining to the oral cavity like caries, oral cancer and oral squamous cell carcinoma, but also those inflicting other body systems like cardiovascular diseases. Certain oral bacteria have also been associated with disorders inflicting the gut like inflammatory bowel diseases and causatively implicated in gut-associated diseases like colorectal cancer. Despite their importance, our knowledge regarding several basis aspects of the oral microbiome (markers of health in oral microbiome, its variation across sub-sites, geography and life-style, oral microbiome members typically associated and the functional microbiome guilds within different sub-sites of the oral microbiome) are limited. A similar observation also pertains to other body sites in the GI-tract as well as skin. Barring a very few studies like the Human Microbiome Project, no study has profiled and investigated the pan-microbiome from different body sites of the same individual. Under this theme, we aim to collate and perform integrated meta-analyses of datasets from other human body sites, including integrated investigation of microbiome data from different body sites sampled from the same individual at a given time-point to understand the pan-human-microbiome interactions.`
    },
];